shared/macros/ansible.jinja

```diff
@@ -188,7 +188,8 @@ value: :code:`Setting={{ varname1 }}`
 #}}
 {{%- macro ansible_sshd_set(msg='', parameter='', value='', config_is_distributed="false", config_basename="00-complianceascode-hardening.conf") %}}
 {{%- if config_is_distributed == "true" %}}
-{{{ ansible_set_config_file_dir(msg, config_file="/etc/ssh/sshd_config", config_dir="/etc/ssh/sshd_config.d", set_file=config_basename, parameter=parameter, separator_regex="\s+", value=value, prefix_regex="^\s*", create='yes', validate='/usr/sbin/sshd -t -f %s', insert_after='', insert_before="^[#\s]*Match") }}}
+{{% set config_dir = "/etc/ssh/sshd_config.d" %}}
```

Comment:
As it is a directory, how putting there a trailing forward slash?

Reply:
I don't know the effect it will have in `config_dir="/etc/ssh/sshd_config.d"` for example.

Reply:
I would need to investigate it

Reply:
The forward slash is not strictly required, it was just a suggestion.
If it will create problem elsewhere we don't need to add it.